connect/eaas/runner/managers/interactive.py

```diff
@@ -30,7 +31,7 @@ def get_method_name(self, task_data, argument):
         return self.handler.events[task_data.input.event_type]['method']
 
     async def get_argument(self, task_data):
-        return task_data.input.data
+        return copy.copy(task_data.input.data)
```

Comment:
[SOL-A] `copy.copy` does a shallow copy. This approach will not fix the problem in the case of deep objects.

It looks like `deepcopy` is more appropriate here.

Reply:
[COD-B] Let's cover this with tests